corehq/apps/style/templates/style/partials/form_list_form.html

```diff
@@ -3,7 +3,7 @@
 {% load i18n %}
 
 <form id="table-form"
-      action="."
+      action="../bootstrap3/partials"
```

Comment:
Curious, what's this? Doesn't this imply that there's a `bootstrap3/partials` directory somewhere in `style` (don't see one)?